src/app/App.js

```diff
@@ -156,7 +158,7 @@ export default function App() {
               <Header title={'Add Feedback'} />
               <FeedbackForm
                 questions={GallupTwelveQuestions}
-                onSubmit={handleFeedbackSubmit}
+                handleSubmit={handleFeedbackSubmit}
```

Comment:
normally you have props name `on`- doing something and in curly brackets you have the `handle`-function. here it would be something like `onSubmit` or `onFeedbackFormSubmit`

Reply:
that's how I call my props, cause otherwise I'd have to pass an onSubmit in the component, when it's actually doing a handleSubmit. The onSubmit is the eventlistener in the  component I thought...